docs/gl_objects/pipelines_and_jobs.rst

```diff
@@ -302,3 +302,27 @@ Play (trigger) a job::
 Erase a job (artifacts and trace)::
 
     build_or_job.erase()
+
+
+List pipeline bridges
```

Comment:
```suggestion
Pipeline bridges
```
It's possible there will be other methods for this later, so maybe we can keep it general and consistent with titles above.